sass/layout/_sidebar-left.scss

```diff
@@ -595,18 +672,25 @@
         /* box-shadow: 0px 3px 4px 0px rgba(0, 0, 0, 0.33);
         transition: box-shadow 0.25s ease-in-out; */
     }
-    .SidebarNavContainer:not(.disabled) .SidebarChannelGroup .SidebarChannelGroupHeader_groupButton:hover {
-        cursor: pointer;
 
+    .SidebarNavContainer:not(.disabled) .SidebarChannelGroup .SidebarChannelGroupHeader:hover {
```

Comment:
I feel it makes the overall CSS more consistent, and it drives more of the actual view logic into styling rather than having it in the component. I'm 4/5 that view logic should be primarily driven by CSS whenever possible, and I don't think we take a huge performance from rendering/not rendering that element.

This allows for specific margins for buttons, though yes there can probably be some common classes here. A lot of this stuff was changed at the last minute, so as mentioned above once everything is in place I'd like to clean up the CSS.